SDLSecurity/SDLPrivateSecurityConstants.h

```diff
@@ -8,11 +8,14 @@
 
 #import <Foundation/Foundation.h>
 
-
+extern NSSet<NSString *> *availableMakes;
```

Comment:
This should be a class property on the below interface

Reply:
Fixed.